packages/desktop/components/SendButton.svelte

```diff
@@ -4,13 +4,16 @@
     import { resetNewTokenTransactionDetails } from '@core/wallet'
     import { openPopup, PopupId } from '@auxiliary/popup'
     import { OnboardingButton } from '@ui'
+    import { SendFlowRouter, sendFlowRouter } from '@views/dashboard/send-flow'
```

Comment:
I'd argue that these files aren't views themselves - they're popups so they should exist alongside the popup components. I'm inclined to say the router code should exist within the `desktop/routers` module but it makes sense alongside the popup components as well.

Reply:
i agree that the router  code should be in `desktop/routers` and the rest in `popups`